index.html

```diff
@@ -28,41 +28,159 @@
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+          <ul class="socials">
+            <li>
+              <a href="#"><img src="./images/socials/twitter.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/socials/Linkedin icon.png" alt=""
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/socials/medium.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/socials/github.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/socials/angellist.png" alt=""
+              /></a>
+            </li>
+          </ul>
 
-        <a href="#" class="arrow-down">
-          <!-- <img src="./images/arrow-down.png" alt="" /> -->
-        </a>
-      </div>
-    </section>
+          <a href="#" class="arrow-down">
+            <!-- <img src="./images/arrow-down.png" alt="" /> -->
+          </a>
+        </div>
+      </section>
+
+      <section class="section-works">
+        <div class="container">
+          <h2 class="secondary-heading">
+            My Recent <br />
+            Works
+          </h2>
+          <p class="separate"></p>
+          <div class="project">
+            <img src="./images/projects/project04.png" alt="" />
+            <h3 class="project-heading">Multi-Post Stories Gain+Glory</h3>
+            <ul class="techs">
+              <li class="techs-item">Ruby on Rails</li>
+              <li class="techs-item">CSS</li>
+              <li class="techs-item">JavaScript</li>
+              <li class="techs-item">HTML</li>
+            </ul>
+
+            <a href="#" class="btn">See project</a>
+          </div>
+
+          <div class="project">
+            <img src="./images/projects/project03.png" alt="" />
+            <h3 class="project-heading">Multi-Post Stories Gain+Glory</h3>
+            <ul class="techs">
+              <li class="techs-item">Ruby on Rails</li>
+              <li class="techs-item">CSS</li>
+              <li class="techs-item">JavaScript</li>
+              <li class="techs-item">HTML</li>
+            </ul>
+
+            <a href="#" class="btn">See project</a>
+          </div>
+          <div class="project">
+            <img src="./images/projects/project02.png" alt="" />
+            <h3 class="project-heading">Multi-Post Stories Gain+Glory</h3>
+            <ul class="techs">
+              <li class="techs-item">Ruby on Rails</li>
+              <li class="techs-item">CSS</li>
+              <li class="techs-item">JavaScript</li>
+              <li class="techs-item">HTML</li>
+            </ul>
+
+            <a href="#" class="btn">See project</a>
```

Comment:
You have done a great work, keep going with it 😄. But you need to fix only one more part 

- [ ]  I think you forgot to implement the `hover` button in this project 
![hover](https://user-images.githubusercontent.com/80458199/227246212-9c86f9d6-5507-4c67-a5da-6a6a98ccaaa8.JPG)

If you check, there is `shadow` when you hover the button, you may check the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=73-469&t=vIV7ct1Ii5QViMJI-0) . Kindly make sure you stuck to the design as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.)